2020-12-02 21:16:28,641 CRITICAL config_server_log.py Критическая ошибка
2020-12-02 21:16:28,650 ERROR config_server_log.py Ошибка
2020-12-02 21:16:28,651 DEBUG config_server_log.py Отладочная информация
2020-12-02 21:16:28,651 INFO config_server_log.py Информационное сообщение
2020-12-02 21:17:27,122 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2020-12-02 21:17:39,809 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2020-12-02 21:18:02,658 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2020-12-02 21:18:44,702 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2020-12-02 21:18:47,283 INFO server.py Установлено соедение с ПК ('127.0.0.1', 59287)
2020-12-02 21:18:47,283 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1606925927.2831836, 'user': {'account_name': 'Guest'}}
2020-12-02 21:18:47,283 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1606925927.2831836, 'user': {'account_name': 'Guest'}}
2020-12-02 21:18:47,284 INFO server.py Cформирован ответ клиенту {'response': 200}
2020-12-02 21:18:47,284 DEBUG server.py Соединение с клиентом ('127.0.0.1', 59287) закрывается.
